z390test/src/test/groovy/org/z390/test/RunAsmTests.groovy

```diff
@@ -0,0 +1,101 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunAsmTests extends z390Test {
+
+    var options = ['trace', 'noloadhigh', "SYSMAC(${basePath("mac")})", "SYSOBJ(${basePath("linklib")})"]
```

Comment:
Why did you add SYSOBJ? See RunAsmDemos.groovy comments.

Reply:
It is added because TESTDC1 and TESTASC1 reference CVTTOHEX module that is assembled as part of bldlib script and the object file is part of linklib.
I can limit the scope to just the above modules as part of the link step that requires access to the object modules.

Reply:
The commit now limits the obj option to those steps that require access to CVTTOHEX obj.